requirements.txt

```diff
@@ -126,7 +127,6 @@ requests-oauthlib==1.3.0
 requests-toolbelt==0.9.1
 rsa==4.7.2
 s3transfer==0.3.4
-scipy==1.5.0
```

Comment:
Do we need scipy as a dependency for something?

Reply:
idk why this got deleted will readd